heritability/import_heritability.py

```diff
@@ -0,0 +1,54 @@
+__author__ = 'Rahul Gupta'
+
+import hail as hl
+
+
+def get_h2_flat_file():
+    return 'gs://ukb-diverse-pops/rg-h2-tables/h2_estimates_all_flat.tsv'
+
+
+def get_h2_ht():
+    return 'gs://ukb-diverse-pops/rg-h2-tables/ht/h2_estimates_all.ht'
+
+
+def import_h2_flat_file(save_to_ht, overwrite):
+    ht = hl.import_table(get_h2_flat_file(), 
+                         delimiter='\t', 
+                         impute=True, 
+                         key=['trait_type','phenocode','pheno_sex','coding','modifier'])
```

Comment:
neither here nor there but we do have this in a global variable somewhere in this repo, like `PHENO_KEY_FIELDS`